config/config-observability.yaml

```diff
@@ -82,3 +82,18 @@ data:
   # the kibana dashboard using `kubectl proxy`.
   logging.revision-url-template: |
     http://localhost:8001/api/v1/namespaces/knative-monitoring/services/kibana-logging/proxy/app/kibana#/discover?_a=(query:(match:(kubernetes.labels.knative-dev%2FrevisionUID:(query:'${REVISION_UID}',type:phrase))))
+
+# METRICS CONFIGURATION
+
+  # metrics.backend-destination field specifies the system metrics destination.
+  # It defaults to prometheus. If this is stackdriver, the metrics will be sent
+  # to stackdriver. "prometheus" and "stackdriver" are supported. This field 
+  # is required.
+  metrics.backend-destination: "prometheus"
+
+  # metrics.stackdriver-project-id field specifies the stackdriver project ID. This
+  # field is optional. When running on GKE, application default credentials will be
+  # used if this field is not provided.
+  # Note: Using stackdriver would incur charges for knative-serving system metrics
+  # before we have own resource types.
```

Comment:
I would simplify this comment to:
`Note: Using stackdriver will incur additional charges`

Reply:
Done